app/build.gradle

```diff
@@ -150,8 +151,10 @@ dependencies {
     /**
      * Test Build Dependencies
      */
-    compile 'com.android.support:appcompat-v7:24.2.1'
-    compile 'com.android.support:recyclerview-v7:24.2.1'
+    compile 'com.android.support:appcompat-v7:26.1.0'
+    compile 'com.android.support:preference-v7:26.1.0'
+    compile 'com.android.support:recyclerview-v7:26.1.0'
+    compile 'com.android.support:design:26.1.0'
```

Comment:
preference-v7 and design are transitive dependencies currently.

com.pavelsikun:material-seekbar-preference uses an old version of preferences and io.apptik.widget:multislider uses an old version of design. 

There's an error in Android Studio warning that using support libraries from different versions could lead to crashes. By including them all libraries use the latest versions. I've tested both seekbars and they work properly on my device